models/util/LuceeORMUtil.cfc

```diff
@@ -27,7 +27,14 @@ component extends="cborm.models.util.ORMUtilSupport" implements="cborm.models.ut
 	 * @return https://docs.jboss.org/hibernate/core/3.5/javadocs/org/hibernate/EntityMode.html
```

Comment:
Can you add the new return type for hibernate 5

Reply:
Added. I also fixed the return type (in both CFORMUtil and LuceeORMUtil) to return the EntityMode class instead of the EntityPersister class.